example/neumann-reduced/heat_equation_reduced.py

```diff
@@ -106,10 +107,14 @@ def advance(self, coupling_output):
 # Create the grid
 dealii.make_grid_and_sparsity_pattern()
 # setup the system, i.e., matrices etc.
-matrix = dealii.create_system_matrix(1, 2, 1.5, 0.5)
+THRESHOLD_X, THRESHOLD_Y = 1.5, 0.5
+matrices = [dealii.create_system_matrix(1, 0, THRESHOLD_X, THRESHOLD_Y),
+            dealii.create_system_matrix(0, 1, THRESHOLD_X, THRESHOLD_Y)]
```

Comment:
@sdrave You selected now for both subregions the same coefficient, but I guess this was intended, right?

Reply:
If I understood the code for `create_system_matrix` correctly, then `THRESHOLD_X`, `THRESHOLD_Y` define a subregion for which the first two parameters define the diffusivitiy in the region and its complement. 

The two parameters for which I have built the model should then be these two diffusivities for a fixed subregion. (Moving the subregion as an additional parameterization would be more difficult.)

Reply:
That's correct. Considering this, you selected for both subregions the coefficient `1` which is then in the end no distinction between both regions. However, this can easily be changed later on.

Reply:
Yes, but these matrices are just summands in a linear combination where the coefficients are the actual diffusivities (1 and 2 in the script).